src/EventService.Models.Dto/Enums/Colour.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace LT.DigitalOffice.EventService.Models.Db.Enums
+{
+  public enum Color
+  {
+    Color1,
```

Comment:
Why you do not do this? 
![image](https://user-images.githubusercontent.com/43515136/215089160-be244724-a13d-4373-aadd-ee28e3b5654c.png)